hummingbot/connector/exchange/bitstamp/bitstamp_constants.py

```diff
@@ -0,0 +1,113 @@
+from hummingbot.connector.constants import MINUTE, SECOND
+from hummingbot.core.api_throttler.data_types import LinkedLimitWeightPair, RateLimit
+from hummingbot.core.data_type.in_flight_order import OrderState
+
+DEFAULT_DOMAIN = ""
+
+REST_URL = "https://www.bitstamp.net/api/"
+WSS_URL = "wss://ws.bitstamp.net"
+
+API_VERSION = "v2"
+
+MAX_ORDER_ID_LEN = None
+HBOT_ORDER_ID_PREFIX = "hbot"
+
+# Order States
+ORDER_STATE = {
+    "Open": OrderState.OPEN,
+    "Finished": OrderState.FILLED,
+    "Expired": OrderState.CANCELED,
+    "Canceled": OrderState.CANCELED,
```

Comment:
there is not state for failed?

Reply:
In the documentation I'm seeing only 'Open, Finished, Expired or Canceled' so if there is one, it's undocumented.